inbm/dispatcher-agent/tests/unit/schedule/test_sqlite_manager.py

```diff
@@ -0,0 +1,100 @@
+from unittest import TestCase
+
+from dispatcher.schedule.sqlite_manager import SqliteManager
+from dispatcher.schedule.schedules import SingleSchedule, RepeatedSchedule
+from dispatcher.dispatcher_exception import DispatcherException
+
+class TestSqliteManager(TestCase):
+    
+    def setUp(self) -> None:
+        self.db = SqliteManager(":memory:")
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** Implement tearDown method to ensure the in-memory database is properly closed after each test to prevent any potential resource leaks.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    def tearDown(self) -> None:
        self.db._conn.close()
```